data/logs_model/logs_producer/splunk_logs_producer.py

```diff
@@ -62,6 +65,15 @@ def __init__(
             raise Exception(
                 "Connection to Splunk refused, check if Splunk instance is available: %s" % e
             )
+        except Exception as e:
+            strict_logging_disabled = config.app_config.get("ALLOW_WITHOUT_STRICT_LOGGING") or (
+                config.app_config.get("ALLOW_PULLS_WITHOUT_STRICT_LOGGING")
```

Comment:
Do we want to set a default value here?

Reply:
it's default value is set in config.py but it's fine to set it again here

Reply:
We shouldn't redefine defaults if we already have them defined in config.py